src/app/models/project.ts

```diff
@@ -13,8 +13,8 @@ export class Project {
     name: string;
     description: string;
     visibility: ProjectVisibility = ProjectVisibility.Private;
-    city: City;
-    scenario: Scenario;
+    city: City = {} as City;
```

Comment:
Yea I couldn't figure out the elegant place for this.
Essentially loading a project with empty city, scenario fields was problematic in the add project workflow. This is a cause of expectations of autocompletes and dropdowns. I can set these fields to {} there, rather than in the model.